modules/local_storage/src/LocalStockChecker.php

```diff
@@ -192,23 +205,21 @@ public function getIsStockManaged(PurchasableEntityInterface $entity) {
    * {@inheritdoc}
    */
   public function getLocationList($return_active_only = TRUE) {
```

Comment:
We are removing the active part of the array but that's OK as the entity holds its status. 
Nice rewrite.